desktop/src/com/unciv/app/desktop/UncivServer.kt

```diff
@@ -12,12 +12,25 @@ import java.io.File
 
 
 internal object UncivServer {
+    private var serverPort = 8080
+
     @JvmStatic
-    fun main(arg: Array<String>) {
+    fun main(args: Array<String>) {
+        args.forEach { arg ->
+            when {
+                arg.startsWith("-port=") -> with(arg.removePrefix("-port=").toIntOrNull() ?: 0) {
+                    if (this in 1.rangeTo(65535)) serverPort = this
+                    else println("'port' must be between 1 and 65535")
```

Comment:
1024 to 49151 should be the range for pets. Below are reserved ports, above are dynamic ports.

Reply:
@yairm210 You don't need to care which one your users use, they use whichever they like

Reply:
I disagree, that's a myopic view. Users who would use my game and encounter an error because they accidentally took a system-dedicated port and it caused them problems *would* open a ticket, and rightly so. Why not avoid problems for other people?

Reply:
Users *will* shot themselves in the foot in any way you allow them to. There is no real benefit to the other ports, anyone who knows enough to know he really needs them would be able to port forward, and so the obvious conclusion is to derby